testCourse/questions/externalGrade/alpine/tests/grade.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/sh
+
+mkdir -p /grade/results
+
+cat /grade/student/answer.txt
+
+if cmp /grade/student/answer.txt <(echo -n "correct"); then
```

Comment:
I know this is mostly for the test suite itself, but if someone is trying this file manually, `cmp` will complain about spaces and blank lines, which shouldn't matter here. So (this is nit), I'd prefer:
```suggestion
if diff -Bwq /grade/student/answer.txt <(echo -n "correct"); then
```

Reply:
Fair, I can make that change, though in my testing it was fine (the editor didn't automatically add spaces or trailing newlines or anything).

Reply:
Done in 4291014.